functions/venue.js

```diff
@@ -462,20 +462,26 @@ exports.toggleDustStorm = functions.https.onCall(async (_data, context) => {
   }
 });
 
+// @debt this is almost a line for line duplicate of exports.updateVenue_v2, we should de-duplicate/DRY these up
 exports.updateVenue = functions.https.onCall(async (data, context) => {
   const venueId = data.id || getVenueId(data.name);
   checkAuth(context);
 
+  // @debt updateVenue_v2 uses checkUserIsAdminOrOwner rather than checkUserIsOwner. Should these be the same? Which is correct?
   await checkUserIsOwner(venueId, context.auth.token.user_id);
```

Comment:
`checkUserIsAdminOrOwner` is the correct one, because the user should be an owner of the admin venue + he should be in the `admins` collection

Reply:
That function is super misleading, as it doesn't do anything extra:

```javascript
const checkUserIsAdminOrOwner = async (venueId, uid) => {
  try {
    return await checkUserIsOwner(venueId, uid);
  } catch (e) {
    return e.toString();
  }
};
```

Reply:
In this case we can remove it later on.

Reply:
This looks more like UI utility function that is supposed to print a message. Probably solved inline in the TSX that calls it. 
Otherwise, suggested type replacement: `boolean|Error` or just an enum having unknown/admin/owner/regular...

Reply:
This is in the backend functions, which are only raw JavaScript at the moment, no types. I'll dig back into the git history sometime in the future to figure where/why it was implemented, but till then, `@debt` suffices.

Reply:
Ended up looking into it now instead..

Looking at the git blame it was introduced in https://github.com/sparkletown/sparkle/pull/1056, at the start of our admin v2 tech debt:

- https://github.com/sparkletown/sparkle/blame/staging/functions/venue.js#L152-L158

So i'm just going to clean it up/remove it in this PR I think.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1485/commits/5df8975f5ddadec52b9f43a1089de3230a39246c